src/cascadia/TerminalApp/TerminalAppLib.vcxproj

```diff
@@ -160,7 +160,7 @@
     <ClCompile Include="Pane.LayoutSizeNode.cpp" />
     <ClCompile Include="ColorHelper.cpp" />
     <ClCompile Include="DebugTapConnection.cpp" />
-    <ClCompile Include="Utils.cpp" />
+    <ClCompile Include="$(OpenConsoleDir)\src\cascadia\inc\Utils.cpp" />

```

Comment:
_hold up, absolutely not_

do not include some other random folder's cpp files

Reply:
**inc is for headers only**

Reply:
It looks like there's only one function that's actually implemented in Utils.cpp, we could _probably_ just move that to the header, and avoid this issue entirely